makefile

```diff
@@ -114,3 +115,9 @@ ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt: ftp.ncbi.nlm.nih.gov/
 # pre-curated by disease. And some of the logic in this Python script is duplicative.
 medgen-xrefs.robot.template.tsv: ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt
 	python src/mondo_robot_template.py -i $< -o $@
+
+# ----------------------------------------
+# SSSOM
+# ----------------------------------------
+medgen.sssom.tsv: ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt
+	python src/create_sssom.py -i $< -o $@ --mapping-namespaces HPO MeSH
```

Comment:
## CLI
I made the default behavior that calling this CLI will produce SSSOM for all namespaces, unless you pass `--mapping-namespaces` / `-m`.